tests/unit/dataactbroker/test_submission_handler.py

```diff
@@ -867,31 +871,35 @@ def test_revert_submission(database, monkeypatch):
               job_type_id=JOB_TYPE_DICT['csv_record_validation'], file_type_id=FILE_TYPE_DICT['appropriations'],
               number_of_warnings=0, number_of_errors=10, filename='new/test/file.csv', number_of_rows=5,
               number_of_rows_valid=0)
-    cert_history = CertifyHistory(submission_id=sub.submission_id)
-    sess.add_all([job, cert_history])
+    pub_history = PublishHistory(submission_id=sub.submission_id)
+    sess.add_all([job, pub_history])
     sess.commit()
 
+    # cert_history = CertifyHistory(certify_history_id=pub_history.publish_history_id, submission_id=sub.submission_id)
```

Comment:
Im guessing this is commented out for the future separation of publishing and certifying. Could it be explained as a TODO here?

Reply:
No, I was just doing something else with it before and forgot to shift it back up after I changed things.